docker/r/Dockerfile

```diff
@@ -63,6 +67,27 @@ RUN chmod +x /tini
 # This allows non-root to install python libraries if required
 RUN mkdir -p ${VENV_DIR} && chown -R ${NB_USER} ${VENV_DIR}
 
+# install miniconda (https://github.com/ContinuumIO/docker-images/blob/master/miniconda3/debian/Dockerfile)
+RUN apt-get update --fix-missing && \
+    apt-get install -y wget bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion && \
```

Comment:
I didn't put them all together so that it would be clear which dependencies belong to which installation -- it's hard enough to read dockerfiles... 

if you think all the apt-get installs should all be in one place, I can add the clarifications in the README -- what do you think?

Reply:
yeah that's true, but it's usually better to keep that all in one layer. At least you don't really need to do the update again since it's done above.